src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -103,7 +103,7 @@ public void LogRequest_SinksToApplicationInsights_ResultsInRequestTelemetry()
                 ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
 
                 HttpMethod httpMethod = GenerateHttpMethod();
-                string host = _bogusGenerator.Company.CompanyName();
+                string host = _bogusGenerator.Company.CompanyName().Replace(" ", "");
```

Comment:
Why is this required? Does it cause any issues?

Reply:
Yes, spaces in host names are now allowed.

Reply:
Let's enforce it in our extension then

Reply:
The extension already uses `HttpRequest` instead so that's already covered, right? The tests only uses this for the stubbing.

Reply:
Does HttpRequest throw an exception for that host name or not? If not, we should add a check in our extension to enforce it maybe?

Reply:
I don't think it guards it very well, so I'll add some extra guards here.

Reply:
Ok thanks!